cmd/lncli/commands.go

```diff
@@ -707,7 +701,41 @@ func closeChannel(ctx *cli.Context) error {
 		req.ChannelPoint.OutputIndex = 0
 	}
 
-	stream, err := client.CloseChannel(ctxb, req)
+	txidChan := make(chan string, 1)
```

Comment:
The following chunk of code is a bit cryptic to a reader I think. Would be nice to add some comments, making it easier to understand the purpose of the go routine and channels.